### PR TITLE
bump up rocm-debug-agent to ROCm 3.8.0

### PR DESCRIPTION
**Pull request by arjun-raj-kuppala**

### CODE REVIEW

File: var/spack/repos/builtin/packages/rocm-debug-agent/package.py [hunk 1]
```diff
@@ -15,23 +15,33 @@ class RocmDebugAgent(CMakePackage):
```
Comment (srekolam) [inline var/spack/repos/builtin/packages/rocm-debug-agent/package.py:15]: you can remove this line

**Review by srekolam — CHANGES_REQUESTED** [review]
i have verified on the centos-8.x and it works.

File: var/spack/repos/builtin/packages/rocm-debug-agent/package.py (hunk 1 above)
Comment (arjun-raj-kuppala) [reply var/spack/repos/builtin/packages/rocm-debug-agent/package.py:15]: Is it this line?. This line is to use cmake template
class RocmDebugAgent(CMakePackage):

am I missing something here?

Comment (srekolam) [reply var/spack/repos/builtin/packages/rocm-debug-agent/package.py:15]: url      = "https://github.com/ROCm-Developer-Tools/rocr_debug_agent/archive/roc-3.5.0.tar.gz"

**Review by srekolam — APPROVED** [review]

**Comment by arjun-raj-kuppala** [comment]
@adamjstewart , can you help merge this

File: var/spack/repos/builtin/packages/rocm-debug-agent/package.py [hunk 2]
```diff
@@ -11,27 +11,36 @@ class RocmDebugAgent(CMakePackage):
     """Radeon Open Compute (ROCm) debug agent"""
 
     homepage = "https://github.com/ROCm-Developer-Tools/rocr_debug_agent"
-    url      = "https://github.com/ROCm-Developer-Tools/rocr_debug_agent/archive/roc-3.5.0.tar.gz"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/rocm-debug-agent/package.py:14]: You should keep this line (and update it to the new URL), it is used by `spack versions` and `spack checksum` to find new versions.

**Comment by arjun-raj-kuppala** [comment]
@adamjstewart , Can you help merge this ?

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**